WordPress/src/main/res/layout/jetpack_badge.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingTop="@dimen/margin_large">
+
+    <com.google.android.material.chip.Chip
```

Comment:
Is there a reason Chip was used here instead MaterialButton?

You could use MaterialButton with cornerRadius like so (just to save you time if you decide to change to Button)
<com.google.android.material.button.MaterialButton
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="center"
        android:paddingTop="6dp"
        android:paddingBottom="6dp"
        android:paddingEnd="15dp"
        android:paddingStart="5dp"
        android:text="@string/wp_jetpack_powered"
        android:textAppearance="?attr/textAppearanceBody1"
        android:textColor="@color/white"
        android:textSize="@dimen/text_sz_large"
        app:backgroundTint="@color/jetpack_badge_background"
        app:cornerRadius="20dp"
        app:iconPadding="10dp"
        app:icon="@drawable/ic_jetpack_logo_24dp"
        tools:ignore="TextContrastCheck" />

Reply:
> Is there a reason Chip was used here instead MaterialButton?

Not really, only for the `cornerRadius`. Thank you for the suggestion. I'll try it out. The chip does something strange with the icon cutting it on the bottom edge and over-brightening the top edge, hence the unexpected `app:chipIconSize="23.99dp"` which is a hack to mitigate the issue.

Reply:
@ravishanker I'd love to replace the `Chip` with a `MaterialButton` but I'm encountering an issue in night mode 🤷 .

Do you have an idea what's wrong?

The code for the `MaterialButton`:
```xml
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:orientation="vertical"
    android:paddingTop="@dimen/margin_large">

    <com.google.android.material.button.MaterialButton
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="center"
        android:paddingEnd="15dp"
        android:paddingStart="5dp"
        android:paddingVertical="5dp"
        android:text="@string/wp_jetpack_powered"
        android:textAppearance="?attr/textAppearanceBody1"
        android:textColor="@color/white"
        android:textSize="@dimen/text_sz_large"
        app:backgroundTint="@color/jetpack_badge_background"
        app:cornerRadius="20dp"
        app:icon="@drawable/ic_jetpack_logo_24dp"
        app:iconPadding="10dp"
        tools:ignore="TextContrastCheck" />
</LinearLayout>
```

and the code for `ic_jetpack_logo_24dp`:
### Light Mode
```xml
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:drawable="@drawable/ic_jetpack_logo_white_24dp" />
</layer-list>
  ```
### Night Mode
```xml
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:drawable="@drawable/ic_jetpack_logo_green_24dp" />
</layer-list>
```

And how it shows:
| Light | Night |
| --- | --- |
| ![light](https://user-images.githubusercontent.com/4588074/178661155-6d0008f7-b2c0-4ca4-bd49-a642f4bbff74.png) | ![night](https://user-images.githubusercontent.com/4588074/178661187-0b792d21-d197-48d3-a6a7-dced5094995c.png) |

Reply:
I managed to make the icon work as intended in night mode by adding `app:iconTint="@null"` to the `MaterialButton` 🎉